plasma_framework/test/src/exits/payment/controllers/PaymentProcessInFlightExit.test.js

```diff
@@ -485,6 +489,39 @@ contract('PaymentProcessInFlightExit', ([_, ifeBondOwner, inputOwner1, inputOwne
             });
         });
 
+        describe('When the Piggyback Bond size is equal to the Exit Bounty', () => {
+            beforeEach(async () => {
+                this.inFlightExitData2 = await buildInFlightExitData(
+                    inputOwner1,
+                    outputOwner1,
+                    ifeBondOwner,
+                    this.piggybackBondSize.toString(),
+                );
+                this.inFlightExitData2.isCanonical = true;
+                await this.exitGame.setInFlightExit(DUMMY_EXIT_ID, this.inFlightExitData2);
+
+                await this.exitGame.setInFlightExitOutputPiggybacked(DUMMY_EXIT_ID, 0);
+            });
+
+            it('should not attempt returning piggyback bond', async () => {
```

Comment:
is it not attempt returning or returning 0??

Reply:
not attempting with the check here https://github.com/omgnetwork/plasma-contracts/pull/707/files/6b5289a0d622372f185649d07211536291ec9a7d#diff-e3ac2ee65e6573c2b59735d9d7c7fd1fb913a7b0833fda900030c4983ff2b286R290